kits/agentic/mockai/PR_DESCRIPTION.md

```
@@ -0,0 +1,10 @@
+## Title: feat(mockai): transform generic app into Mockai Interview prep platform
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)